example/gluon/word_language_model/get_ptb_data.sh

```diff
@@ -17,6 +17,11 @@
 # specific language governing permissions and limitations
 # under the License.
 
+echo ""
+echo "NOTE: Please review the licensing of the datasets in this script before proceeding"
+echo "See https://catalog.ldc.upenn.edu/ldc99t42 for the licensing"
+echo "Once that is done, please uncomment the wget commands in this script"
```

Comment:
How about we print these message and let users choose Y/N

Reply:
We can do that, but for now I think the important thing is to merge this into master, since this is already part of 1.0 and was done this way based on Apache guidelines.

Reply:
The next release is at least a month away.
This should be fairly quick. We can wait.

Reply:
Sure. Updated across all usages of PTB license.